.github/workflows/static.yml

```diff
@@ -0,0 +1,43 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
```

Comment:
metadata.name
the pod's name
metadata.namespace
the pod's [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces)
metadata.uid
the pod's unique ID
metadata.annotations['<KEY>']
the value of the pod's [annotation](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations) named <KEY> (for example, metadata.annotations['myannotation'])
metadata.labels['<KEY>']
the text value of the pod's [label](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels) named <KEY> (for example, metadata.labels['mylabel'])
The following information is available through environment variables but not as a downwardAPI volume fieldRef:

spec.serviceAccountName
the name of the pod's [service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/)
spec.nodeName
the name of the [node](https://kubernetes.io/docs/concepts/architecture/nodes/) where the Pod is executing
status.hostIP
the primary IP address of the node to which the Pod is assigned
status.podIP
the pod's primary IP address (usually, its IPv4 address)
The following information is available through a downwardAPI volume fieldRef, but not as environment variables:

metadata.labels
all of the pod's labels, formatted as label-key="escaped-label-value" with one label per line
metadata.annotations
all of the pod's annotations, formatted as annotation-key="escaped-annotation-value" with one annotation per line
Information available via resourceFieldRef
These container-level fields allow you to provide information about [requests and limits](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) for resources such as CPU and memory.

resource: limits.cpu
A container's CPU limit
resource: requests.cpu
A container's CPU request
resource: limits.memory
A container's memory limit
resource: requests.memory
A container's memory request
resource: limits.hugepages-*
A container's hugepages limit
resource: requests.hugepages-*
A container's hugepages request
resource: limits.ephemeral-storage
A container's ephemeral-storage limit
resource: requests.ephemeral-storage
A container's ephemeral-storage request